app/commands/new_data_category_command.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class NewDataCategoryCommand < ApplicationCommand
+  usage do
+    no_command
+  end
+
+  keyword :name do
+    desc "Category name to create"
+    required
+  end
```

Comment:
Looks like these change how data is passed to the command. Want to add an example usage to your PR description so this is highlighted?

Old method: `--name="foo"`, new method: `NAME="foo"`

Reply:
Yes intentional — haven't documented any of these commands yet, and figure they'll change again as we net out a few more of these

Reply:
I'll make sure we have proper docs for these though — after which changes will require update to those 😁